lib/src/sql/part.rs

```diff
@@ -29,6 +29,7 @@ pub enum Part {
 	Where(Value),
 	Graph(Graph),
 	Value(Value),
+	Start(Value),
```

Comment:
I think a doc-comment would be useful on this and other variants, describing the purpose and/or giving an example. I only understood the meaning of `Start` after reading half the PR.

Reply:
Ok cool, I'll put comments in a separate PR 👍 .